build.rs

```diff
@@ -253,9 +252,6 @@ fn make_elfutils(compiler: &cc::Tool, src_dir: &path::PathBuf, out_dir: &path::P
             }
         })
         .collect();
-
-    #[cfg(target_arch = "aarch64")]
```

Comment:
Put the aarch64 kludge back in please

Reply:
Ah okay. I should not have reverted that change then. Ignoring it now.